ViewModel/Saving/PersistenceManagers/AlternativeComparisonReportPersistenceManager.cs

```diff
@@ -0,0 +1,88 @@
+﻿using FdaLogging;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using ViewModel.AlternativeComparisonReport;
+using ViewModel.Utilities;
+
+namespace ViewModel.Saving.PersistenceManagers
+{
+    public class AlternativeComparisonReportPersistenceManager : SavingBase, IElementManager
+    {
+        private const int XML_COL = 2;
+        public override string TableName => "alternative_comparison_reports";
+
+        public override string[] TableColumnNames => new string[] { NAME, "xml" };
+
+        public override Type[] TableColumnTypes => new Type[] { typeof(string), typeof(string) };
+
+        public AlternativeComparisonReportPersistenceManager(Study.FDACache studyCache)
+        {
+            StudyCacheForSaving = studyCache;
+        }
+
+        public override ChildElement CreateElementFromRowData(object[] rowData)
+        {
+            string xml = (string)rowData[XML_COL];
```

Comment:
Do an index check, this could be out of bounds

Reply:
Also cast check. you can do:
if (rowData[XML_COL] is string xml)
{
...use xml as variable
}

Reply:
Cool. I like it. Done.